doc/docs.md

```diff
@@ -583,8 +583,8 @@ arr := []int{len: 5, init: -1}
 // `[-1, -1, -1, -1, -1]`
 ```
 
-Setting the capacity improves performance of insertions,
-as it reduces the number of reallocations needed:
+Setting the capacity improves performance of insertions since it will preallocate some memory   
+so it will reduce the number of reallocations needed:
```

Comment:
ok

Reply:
`Setting the capacity improves performance of insertions as it will preallocate memory and reduce the number of reallocations needed: